src/components/FiltersDrawer.tsx

```diff
@@ -154,20 +149,29 @@ export const FiltersDrawer = ({ open, onClose }: FiltersDrawerProps) => {
             value={filters?.business_market}
             options={transformOptions(businessMarketMap)}
             onSelect={(business_market) => setFilters((prev) => ({ ...prev, business_market }))}
+            clearValue={() => setFilters((prev) => ({ ...prev, business_market: undefined }))}
           />
         </div>
-        <Button
-          type='primary'
-          className='filters-cta'
-          onClick={() => {
-            dispatch({
-              type: 'filters',
-              payload: filters,
-            })
-            handleClose()
-          }}>
-          Apply Filters
-        </Button>
+        <div className='filters-cta-container'>
+          <Button
+            type='text'
+            className='filters-cta clear'
+            onClick={() => {
+              dispatch({ type: 'clearFilters' })
+              handleClose()
```

Comment:
I don't like this behaviour of closing when a user presses clear filters, I think it's more user friendly if we keep the sidebar open as the user would likely add a new set of filters instead of just searching with no filters at all

Reply:
I agree, didn't think about it so much, it was 1-2 am when I was working on this 😂